lib/src/driver.dart

```diff
@@ -219,16 +223,24 @@ class Driver {
   }
 
   /// Decide on the appropriate method for resolving URIs based on the given
-  /// [packageRootPath] and [customUrlMappings] settings, and return a
+  /// [options] and [customUrlMappings] settings, and return a
   /// [SourceFactory] that has been configured accordingly.
   SourceFactory _chooseUriResolutionPolicy(
-      String packageRootPath, Map<String, String> customUrlMappings) {
+      CommandLineOptions options, Map<String, String> customUrlMappings) {
     List<UriResolver> resolvers = [
       new CustomUriResolver(customUrlMappings),
       new DartUriResolver(sdk)
     ];
-    if (packageRootPath != null) {
-      JavaFile packageDirectory = new JavaFile(packageRootPath);
+
+    if (options.packageConfigPath != null) {
+      // TODO(pquitslund): plug into new resolver once implemented
+      // https://github.com/dart-lang/sdk/issues/23615
+      // Uri fileUri = new Uri.file(options.packageConfigPath);
+      // File configFile = new File.fromUri(fileUri);
+      // List<int> bytes = configFile.readAsBytesSync();
+      // Map<String, Uri> map = pkgfile.parse(bytes, fileUri);
```

Comment:
We should probably print an error message and fail rather than pretend that everything is OK when we're really going to ignore the file.

Similarly, some or all of the commented out code above will need to be in a try-catch so that we can report an error if the file cannot be read. (And there's probably no reason not to uncomment the code at this point.)